pkg/clusterbuilder/providers/capa/values/cluster_values.yaml

```diff
@@ -20,9 +20,10 @@ global:
       maxSize: 5
       minSize: 2
       rootVolumeSizeGB: 25
-      spotInstances:
-        enabled: true
-        maxPrice: 0.2960
+      # spot instances are disabled for china because the pricing there is so random that it is hard to set any reliable values that will not cause e2e flaky
+      #spotInstances:
+      #  enabled: true
```

Comment:
You'll want to specifically set this to false to avoid any potential issues when overlaying values files on top of each other

Reply:
ACK

Reply:
fixed

Reply:
Uh I need separate values file

Reply:
Yeah exactly. See here for example: https://github.com/giantswarm/cluster-standup-teardown/blob/6c8ccfb679a178750107cbc7e4425a8815ac24cd/pkg/clusterbuilder/providers/capa/managed_controlplane_cluster.go#L14-L15